Test/Common/DomainTest/implementation/VolunteerOpportunityTest.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Ttu.Domain;
+
+namespace Ttu.DomainTest.implementation
+{
+    [TestClass]
+    public class VolunteerOpportunityTest
+    {
+        private VolunteerOpportunity _volunteerOpportunity;
+
+        private static readonly User TestUser = new User()
+        {
+            Description =  "description",
+            EmailAddress = "email",
+            FirstName = "First",
+            LastName = "Last",
+            Location = "here",
+            PasswordEncrypted = "Password",
+            RecordId = 500,
+            UserId = "8000"
+        };
+
+        private static readonly Organization TestOrganization = new Organization()
```

Comment:
I would just make this a helper method and not make it static (you might need to create an instance variable depending on how you implement it). Then you can just initialize it in the SetUp.